packages/api/src/controllers/stream.test.js

```diff
@@ -373,8 +373,8 @@ describe("controllers/stream", () => {
       expect(res.status).toBe(403);
     });
 
-    it("should not get all streams for admin user", async () => {
-      client.apiKey = adminApiKey;
+    it("should not get all streams for non admin user", async () => {
```

Comment:
@darkdarkdragon changed the test to test for `non admin user`.

It's a bit weird, the `authMiddleware({ admin: true })` creates the need for a JWT, but the endpoint as is doesn't require it. Do you think I should update the permissions for it to require a JWT? 🤔

Reply:
> Do you think I should update the permissions for it to require a JWT?

No, there is no need.